site/src/pages/WorkspacePage/WorkspaceDeleteDialog/WorkspaceDeleteDialog.tsx

```diff
@@ -0,0 +1,182 @@
+import { Workspace, CreateWorkspaceBuildRequest } from "api/typesGenerated";
+import { useId, useState, FormEvent } from "react";
+import { ConfirmDialog } from "components/Dialogs/ConfirmDialog/ConfirmDialog";
+import { type Interpolation, type Theme } from "@emotion/react";
+import { colors } from "theme/colors";
+import dayjs from "dayjs";
+import TextField from "@mui/material/TextField";
+import { docs } from "utils/docs";
+import Link from "@mui/material/Link";
+import Checkbox from "@mui/material/Checkbox";
+
+const styles = {
+  workspaceInfo: (theme) => ({
+    display: "flex",
+    justifyContent: "space-between",
+    backgroundColor: colors.gray[14],
+    border: `1px solid ${theme.palette.divider}`,
+    borderRadius: 8,
+    padding: 12,
+    marginBottom: 20,
+    lineHeight: "18px",
```

Comment:
For the line height here, I feel like it'd be better to use a relative value (and because it's line height, a value with no units), so that the UI behaves consistently if the user needs to zoom their screen in

Reply:
You know, I tried this at first but because the relative value of the line-height is multiplied by the font-size, and because we have different font-sizes in the container, we get uneven line height:

![Screenshot 2023-11-13 at 1 42 44 PM](https://github.com/coder/coder/assets/19142439/b6ce1cfb-7705-448a-b10b-286bb66eae3a)

I tested on different screen sizes and we run into flex box issues before we ever run into line-height issues, so I figured I'd just go with px. Is there an easier path forward I'm missing?  I'd prefer not to specify a bunch of different line-heights.

Reply:
I'm not fully sure off the top of my head, but I *think* that using `em` units (instead of unit-less or `rem`) might do the trick?

[It's talked about a little here](https://stackoverflow.com/a/12304722)